Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -284,14 +283,22 @@ def read_info_file(filename):
             demisto.info(f"File '{filename}' contents: {ret_value}.")
             return ret_value
     except FileNotFoundError:
+        demisto.info(f"File '{filename}' does not exist")
         return None
 
 
-def write_info_file(filename, contents):
+def write_info_file(filename, contents, overwrite=False):
     demisto.info(f"Saving File '{filename}' with {contents}.")
-    with open(filename, 'w') as file:
-        file.write(f"{contents}")
-        demisto.info(f"File '{filename}' saved successfully with {contents}.")
+    mode = 'w' if overwrite else 'a'
+    try:
+        with open(filename, mode, encoding='utf-8') as file:
+            if overwrite:
+                file.write(contents)
+            else:
+                file.write(f"\n{contents}")
+            demisto.info(f"File '{filename}' saved successfully with {contents}.")
+    except (IOError, OSError) as e:
```

Comment:
Why not catch any Exception?
```suggestion
    except Exception as e:
```